FF13Rando/bin/data/enemies.csv

```diff
@@ -147,6 +147,7 @@ m213,Svarog,Flying,14
 m214,Circuitron,,5
 m215,Cryohedron,,10
 m216,Juggernaut,,25
+m222,Syphax,,26
```

Comment:
So Syphax was removed from the list to remove it from the pool. I'll add a trait to remove this from the pool though. idk why I didn't do that before